dev/integration_tests/module_host_with_custom_build/app/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 28
+    compileOptions {
+        sourceCompatibility 1.8
+        targetCompatibility 1.8
+    }
+    defaultConfig {
+        applicationId "io.flutter.add2app"
```

Comment:
Let's globally use add-to-app instead of add2app generally.

Reply:
and for packages that would be "addtoapp" to avoid special characters

Reply:
Matt is correct: https://developer.android.com/studio/build/application-id